### PR TITLE
Hide range inputs on IE9 in pixelation widget.

### PR DESCRIPTION
**Pull request by davidsbailey**
Chrome:
screen shot 2015-07-28 at 2 08 21 pm

IE9, before:
screen shot 2015-07-28 at 2 10 04 pm

IE9, after:
screen shot 2015-07-28 at 1 59 42 pm

### CODE REVIEW

File: dashboard/public/pixelation/pixelation.css [hunk 1]
```diff
@@ -102,3 +102,8 @@ input[type=range] {
 .label_group label {
   display: inline-block;
 }
+
+/* IE9 renders range inputs as type text. Hide the input in this case. */
+input[type="text"].rangeInput {
+  display: none;
+}
```
Comment (joshlory) [inline dashboard/public/pixelation/pixelation.css]: Nit: no final newline.

**Comment by joshlory** [comment]
LGTM

**Comment by joshlory** [comment]
Wait, how does this target IE9 only?

**Comment by davidsbailey** [comment]
IE9 rewrites inputs of type "range" as type "text". so, the inputp[type="text"] part of the rule is only activated on browsers that do this rewrite. the only one of these browsers that we know/care about is IE9. Let me know if I can improve the comment!

**Comment by joshlory** [comment]
Ah I see, I didn't know IE9 rewrote type="range" -> "text".  Makes sense!

**Pull request merged**